src/templates/calendar/calendar.js

```diff
@@ -0,0 +1,136 @@
+import { React, useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import dayjs from 'dayjs';
+import localeDa from 'dayjs/locale/da'; // With a custom alias for the locale object
+import localizedFormat from 'dayjs/plugin/localizedFormat';
+import { IntlProvider, FormattedMessage } from 'react-intl';
+import './calendar.scss';
+
+/**
+ * Slideshow component.
+ *
+ * @param {object} props
+ *   Props.
+ * @param {object} props.content
+ *   The slide content.
+ * @returns {JSX.Element}
+ *   The component.
+ */
+function Calendar({ content }) {
+  dayjs.extend(localizedFormat);
+  const [translations, setTranslations] = useState();
+  const { backgroundColor, hasDateAndTime, title, events } = content;
+  const classes = `template-calendar ${backgroundColor}`;
+
+  /**
+   * Imports language strings.
+   */
+  useEffect(() => {
+    import('./lang/da.json').then((data) => {
+      setTranslations(data);
+    });
+  }, []);
+
+  // Sort events by datetime and filter away events that are done.
+  const sortedEvents = events
+    .filter((e) => {
+      return new Date(e.datetime).getTime() > new Date().getTime();
```

Comment:
This should also filter away events take place after the event, if this is a mirror of the "manual-calendar".
Perhaps we should have a "only-present-day" boolean to turn this behaviour on/off.